ci/build_docs.sh

```diff
@@ -52,18 +54,23 @@ pushd cpp/doxygen
 doxygen Doxyfile
 mkdir -p "${RAPIDS_DOCS_DIR}/libcugraph/html"
 mv html/* "${RAPIDS_DOCS_DIR}/libcugraph/html"
+mkdir -p "${RAPIDS_DOCS_DIR}/libcugraph/_xml"
+mv xml/* "${RAPIDS_DOCS_DIR}/libcugraph/_xml"
```

Comment:
```suggestion
```

These lines can be removed for the following reasons:

- the libcugraph HTML files are no longer being used (only the XML is used)
- the XML files don't need to be moved to the `RAPIDS_DOCS_DIR` directory
  - moving them to that directory seems to break the builds as indicated by this preview page: https://downloads.rapids.ai/ci/cugraph/pull-request/3938/2b88734/docs/cugraph/html/api_docs/cugraph_c/community/